docs/builders/tart.mdx

```diff
@@ -13,10 +13,10 @@ The `tart` builder is used to create macOS and Linux VMs for Apple Silicon power
 
 Here are some highlights of Tart:
 
-* Tart uses Apple's own `Virtualization.Framework` for [near-native performance](https://browser.geekbench.com/v5/cpu/compare/14966395?baseline=14966339).
-* Push/Pull virtual machines from any OCI-compatible container registry.
-* Built-in CI integration.
-* Use this Tart Packer Plugin to automate VM creation.
+- Tart uses Apple's own `Virtualization.Framework` for [near-native performance](https://browser.geekbench.com/v5/cpu/compare/14966395?baseline=14966339).
```

Comment:
Why change that? 🤔

Reply:
Sorry, it was automatic formatter for Markdown files (and since I included other formatting changes for Go files I figured I can as well leave it in too). Happy to revert if you'd like!